web/src/Root.jsx

```diff
@@ -70,6 +71,9 @@ const Header = () => {
 };
 
 const Sidebar = () => {
+  const navigate = useNavigate();
+  const { products } = useProduct();
+
```

Comment:
Now or in the future, I think la the logic for displaying the `ChangeProductButton` should live in it's own component.